easybuild/tools/options.py

```diff
@@ -131,6 +131,7 @@ def terminal_supports_colors(stream):
 DEFAULT_SYS_CFGFILES = [sorted(glob.glob(os.path.join(d, 'easybuild.d', '*.cfg')))
                         for d in XDG_CONFIG_DIRS]
 DEFAULT_USER_CFGFILE = os.path.join(XDG_CONFIG_HOME, 'easybuild', 'config.cfg')
+DEFAULT_LOCAL_CFGFILE = os.path.join(os.path.normpath(os.getcwd()), 'config.cfg')
```

Comment:
Maybe rename to `DEFAULT_CWD_CFGFILE`, or `DEFAULT_CURRENT_WORKING_DIR_CFGFILE` if that's not clear enough?
I think "local" could be interpreted differently